src/components/buttons/button.css

```diff
@@ -0,0 +1,43 @@
+.buttonBox {
+  width: 40%;
+  height: calc(100% - 110px);
+  display: grid;
+  grid-template-columns: repeat(4, 3fr);
+  grid-template-rows: repeat(4, 1fr);
+  grid-gap: 1px;
+}
+
+button:nth-child(4n+0) {
+  background-color: rgb(238, 151, 20);
+}
+
+.display {
+  color: white;
+  display: flex;
+  background-color: rgb(150, 150, 150);
+  justify-content: end;
+  width: 40%;
+}
+
+button {
+  border: none;
+  background-color: rgb(185, 185, 185);
+  height: 40px;
+  font-size: 15px;
+  color: rgb(17, 16, 16);
+  font-weight: bold;
+  cursor: pointer;
+  outline: none;
+}
```

Comment:
- [ ] Good job styling the calculator but you are required to follow the design but your `=` is too large while the `0` underneath is small. Kindly rectify this.
- [ ] (OPTIONAL) It would be great if you could give width and height to your calculator so that it can look good on all screen sizes 👍 
- Your design:
![Screenshot 2022-06-29 184052](https://user-images.githubusercontent.com/85113567/176502286-52548cee-b51d-4ca0-a138-5c515a05031b.png)
- Required design:
![Screenshot 2022-06-29 184115](https://user-images.githubusercontent.com/85113567/176502318-013b36f8-e026-40d7-a9ba-6de1e4a48bd1.png)

Reply:
@Kingstalux Thank you for the suggestions, your right. Now it looks better!.